frontend/app/forms/motion-to-waive-fees/motion-to-waive-fees.web.component.js

```diff
@@ -1,23 +1,580 @@
+              dataKey="person.income.childSupport"
+            />
+            <TextInput
+              label="Payments from civil litigation "
+              dataKey="person.income.civilPayments"
+            />
+            <TextInput
+              label="Victim restitution"
+              dataKey="person.income.victimRestitution"
+            />
+            <TextInput
+              label="Public assistance"
+              dataKey="person.income.publicAssistance"
+            />
+            <small className="web-form-input">
+              (Including AFDC, FEP, TANF, welfare, etc.)
+            </small>
+            <TextInput
+              label="Financial support from household members"
+              dataKey="person.income.householdFinancialSupport"
+            />
+            <TextInput
+              label="Financial support from non-household members"
+              dataKey="person.income.otherFinancialSupport"
+            />
+            <TextInput
+              label="Trust income"
+              dataKey="person.income.trustIncome"
+            />
+            <TextInput
+              label="Annuity income"
+              dataKey="person.income.annuityIncome"
+            />
+          </>
+        )}
+        <div className="web-form-input">Total:</div>
+        <CheckBox
+          label="I have no income because:"
+          dataKey="person.noGrossMonthlyIncome"
+          name="grossMonthlyIncome"
+        />
+        {data.person.noGrossMonthlyIncome && (
+          <TextArea
+            label="Explain"
+            dataKey="person.noGrossMonthlyIncome.explanation"
+          />
+        )}
+      </Section>
+
+      <Section name="6. Monthly Tax Deductions">
+        <CheckBox
+          label="I have the following monthly tax deductions."
+          dataKey="person.monthlyTaxDeductions"
+          name="monthlyTaxDeductions"
+        />
+        {data.person.monthlyTaxDeductions && (
+          <>
+            <TextInput
+              label="Federal income tax"
+              dataKey="person.tax.federal"
+            />
+            <TextInput label="State income tax" dataKey="person.tax.state" />
+            <TextInput
+              label="Municipal income tax"
+              dataKey="person.tax.municipal"
+            />
+            <TextInput label="FICA" dataKey="person.tax.FICA" />
+            <TextInput label="Medicare" dataKey="person.tax.medicare" />
+          </>
+        )}
+        <CheckBox
+          label="I have no monthly tax deductions because I have no income."
+          dataKey="person.noMonthlyTaxDeductions"
+          name="monthlyTaxDeductions"
+        />
+      </Section>
+
+      <Section name="7. Monthly expenses">
+        <small className="web-form-input">
+          (Include amounts you pay for yourself and any spouse, children or
+          other dependents in your household)
+        </small>
+        <TextInput label="Rent or mortgage" dataKey="person.income.rent" />
```

Comment:
We have `person.income.rentalIncome` and then we have `person.income.rent`.

This appears to be in the expenses section. Should it be `person.expenses.rent`? Same goes for all the other data keys in this section